src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -352,6 +359,38 @@
      * @return an instance of MixpanelAPI associated with your project
      */
     public static MixpanelAPI getInstance(Context context, String token) {
+        return getInstance(context, token, false);
```

Comment:
have we made corresponding documentation changes?

Reply:
if the end user opted out, shutdown the app and restart. now when `getInstance(Context context, String token)` gets called, looks like we'll set `optOutTrackingDefault` to `false`? how does it work in that case?

Reply:
that's just the default flag - it won't override user's action